CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0] - 2019-09-05
```

Comment:
I don't see the `0.2.0` tag anywhere, or a commit referencing it. Should these changes instead be labelled as `Unreleased`?

Reply:
I will tag it in Github. It is used in the `setup.py` file.

https://github.com/slazicoicr/dashi/blob/f87dd16e6ff00f2d8db322143755c6467177b9f8/setup.py#L5

I still want to keep specific versions, as it makes it easy to install a specific pre-release version for development.

Reply:
That makes sense. We often do a specific commit just for the release so that it can easily be referenced in the history; for this project, if you were to do so, the only code change in that commit would be the version number in setup.py.